packages/card/event-card/index.tsx

```diff
@@ -0,0 +1,66 @@
+import Card from '../card'
+import { CardProps, ProductBadgesProps } from '../types'
+import s from './style.module.css'
+
+interface EventCardProps {
+  appearance?: CardProps['appearance']
+  link: string
+  eventDate: string
+  eventType: string
+  title: string
+  productBadges?: ProductBadgesProps['badges']
+}
+
+export function EventCard({
+  appearance,
+  link,
+  eventDate,
+  eventType,
+  title,
+  productBadges,
+}: EventCardProps) {
+  const ctaText = `Register for ${eventType.toLowerCase()}`
+
+  return (
+    <Card heading={title} link={link} appearance={appearance}>
+      <div className={s.eventCardInner}>
+        <Card.Content>
+          <Card.Meta items={[eventDate, eventType]}></Card.Meta>
+          <Card.Heading>{title}</Card.Heading>
+          {productBadges && productBadges?.length > 0 ? (
+            <Card.ProductBadges
+              badges={productBadges}
+              appearance={appearance}
+            />
+          ) : null}
+          <span className={s.pseudoCta}>
```

Comment:
Assuming this pseudoCta should be anchored to the bottom of the card. Maybe something to confirm with design.

![CleanShot 2022-10-25 at 11 32 12@2x](https://user-images.githubusercontent.com/825855/197817389-7ec260eb-e3dd-40ac-b219-1d1602b33b85.png)

Reply:
Might have to reconsider withArrow functionality to make this happen 🤔

Reply:
I still need to bring over the `withArrow` prop/logic from #777, once I do the card will match spec and I'll commit your other suggestion (thanks for that)

Reply:
Right, but the pseudoCta wont be able to be bottom aligned since the cta div is still in the document and taking up the space at the bottom of the card.

Reply:
Hmm...it looks like it's flush with `<Content />`'s padding of `24px`, now that I've updated the card with `withArrow` 🤔 

![chrome_1L9nFpnMMr](https://user-images.githubusercontent.com/3746694/197823640-e69e2273-68f8-4cb3-9db5-4580697848be.png)

Reply:
We discussed this and I hadn't tested with different event title content, so I never noticed how there was no guaranteed alignment of the pseudo-CTA with the bottom of the card. 

This issue was addressed in this commit:
https://github.com/hashicorp/react-components/pull/780/commits/1627dfb1758bd318d41fa1d47378f30b3e072595